XCUITests/FxScreenGraph.swift

```diff
@@ -540,6 +561,12 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> Scree
         screenState.tap(rdsButton, forAction: Action.ToggleRequestDesktopSite) { userState in
             userState.requestDesktopSite = !userState.requestDesktopSite
         }
+
+        let trackingProtectionButton = app.sheets.element(boundBy: 0).buttons.element(boundBy: 1)
+        let tpIsOn = "trackingProtectionSetting == \(FxUserState.TrackingProtectionSetting.alwaysOn.rawValue) || (trackingProtectionSetting == \(FxUserState.TrackingProtectionSetting.privateOnly.rawValue) && isPrivate == YES)"
```

Comment:
Maybe a better way? NSPredicates trip me up (in this case due to the use of Enum) and this is what I end up with.

Reply:
Do you mean a) FxUserState gets a member `let trackingProtectionOnCondition = "the predicate string"`? Makes sense to encapsulate it there and make the enum private yeah.

And for b) I don't know where else it would go (only userstate seems to be the place to put it).

Reply:
a. Hadn't thought of putting the predicate into the enum value itself. Not sure if I feel dirty doing that :O) I was only really talking about making the enum fileprivate.
b. If it was private, it could go right next to the `FxUserState`. It wouldn't be exposed to the tests, (or anything outside of `FxScreenGraph`). Anything to make it less verbose.

Reply:
Much nicer making the enum fileprivate at top-level, updating and then landing, thanks!